bluechi.spec.in

```diff
@@ -3,6 +3,10 @@
 %global with_python 1
 %endif
 
+# coverage collection is disabled by default , it can be enabled passing `--define "with_coverage 1"` option to rpmbuild
+%if 0%{!?with_coverage:0}
+%global coverage_flags -Dwith_coverage=0
+%endif
```

Comment:
Ah, sorry, only now realized that we could simplify this if-block to:

```
# Coverage collection is disabled by default , it can be enabled passing `--define "with_coverage 1"` option to rpmbuild
%global coverage_flags -Dwith_coverage=%{?with_coverage:0}
```

Reply:
But with_coverage variable is supposed to be true or false.  Should I change it in the meson to be 0 or 1 instead of true or false?

Reply:
Ah, forgot about it, so then we need to use longer code:

```
# Coverage collection is disabled by default , it can be enabled passing `--define "with_coverage 1"` option to rpmbuild
%if 0%{!?with_coverage:0}
%global coverage_flags -Dwith_coverage=true
%endif
```